xplique/attributions/base.py

```diff
@@ -53,8 +56,16 @@ def __init__(self, model: Callable, batch_size: Optional[int] = 64):
             if model_key not in BlackBoxExplainer._cache_models:
                 BlackBoxExplainer._cache_models[model_key] = model
             self.model = BlackBoxExplainer._cache_models[model_key]
+            self.inference_function = predictions_one_hot
+            self.batch_inference_function = batch_predictions_one_hot
+        elif isinstance(model, (tf.Module, tf.keras.layers.Layer)):
```

Comment:
Ok so Sequential Keras.Model and stuff inherit from tf.keras.layers.Layer ? :)

Reply:
Sequential inherit from tf.keras.Model so it goes into the first if! tf.keras.layers.Layer and tf.Module are a little bit different. I believe though that tf.keras.model inherit from tf.keras.Model so it is import to use the elif after the first one!